ZombieMonster/TextScripts/JumpScript

AyeEmWhy opened the pull request.

AyeEmWhy committed:
included jumping script  + zombieMonster

ISSUE: zombieMonster script does not move 
      --- zombieMonster is not yet a preFab
cleaned up some of the folders 
fixed jumping issue for player

AyeEmWhy committed:
add additional character scripts

AyeEmWhy committed:
Delete mainCharacter.prefab

AyeEmWhy committed:
Delete mainCharacter.prefab.meta

AyeEmWhy committed:
Delete First Scene.unity

AyeEmWhy committed:
Delete First Scene.unity.meta

AyeEmWhy committed:
include zombie AI + death effect

=> also includes player knock-back when  enemy zombie AI detects collision with player.
=> Added in a HealthManager Script for all enemies

AyeEmWhy committed:
delete to reupload sorted folders

AyeEmWhy committed:
del scripts for reupload

AyeEmWhy committed:
included sorted folders

AyeEmWhy committed:
added in Dialogue/TextScripts + sort Folders

=> fixed issue with zombieMovementController.cs causing error when interacting with pencil shooter.
=> Text Scripts include:
      - ActivateText
     - FlashingText
     -TextBoxManager

deathman1 commented:
Thanks


On Jun 30, 2017 11:46 PM, "Amy Jiang" <notifications@github.com> wrote:

> @AyeEmWhy <https://github.com/ayeemwhy> requested your review on:
> so0p/unity2dgame#7 <https://github.com/so0p/unity2dgame/pull/7>
> ZombieMonster/TextScripts/JumpScript.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/so0p/unity2dgame/pull/7#event-1146871799>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AWBA4_BtRozQDRBbOZeU9y1og84B3VwRks5sJesfgaJpZM4OLPHk>
> .
>

AyeEmWhy committed:
added in code to destroy DeathEffect after 2 sec

=> included BloodSplatter in preFab->enemyPreFabs (folder)
=> included DestroyBloodSplatter Script in EnemyScripts->SharedEnemyScripts (folder)

AyeEmWhy committed:
commit changes to proj. settings

AyeEmWhy committed:
updating to current

AyeEmWhy committed:
update to work with current

AyeEmWhy committed:
update

AyeEmWhy committed:
Add files via upload
